### DISCUSSION TITLE
In Release `v1.12.0`, it says "Use nftables as default driver for Fedora 41", what about debian?

### DISCUSSION THREAD

**Discussion by TurnOffNOD** [containers/netavark#1050] [General]
Hi, in Release `v1.12.0`, the release info says `Use nftables as default driver for Fedora 41`, what about Debian? Is nftables default driver in release `v1.12.0` for debian?

**Comment by Luap99** [comment]
No we do not control debian packaging at all. Changing the default driver is a breaking change so we will not change the upstream default without a major version.

However there is a build option so any distro can choose whatever default they like
https://github.com/containers/netavark/blob/93a821adaba95ef9d9e129e5ee1a55726865297e/build.rs#L69
As such we went ahead and set the default to nftables for fedora as we think it is the better option now
https://github.com/containers/netavark/commit/097c32fb7fa5268005434fbe0bbc42fafac53cd8

I would recommend other distros to switch over to nftables as well but it is a breaking change so it should only be done for major distro version updates IMO.

**Reply by mheon** [reply]
To be clear, this is breaking in the sense that it will effectively require a system restart to get container networking running properly again - not in the sense that existing containers will no longer have networking.
